depclean-maven-plugin/src/main/java/se/kth/depclean/util/JarUtils.java

```diff
@@ -51,14 +51,16 @@ public static void decompressJars(final String outputDirectory) {
     for (File f : Objects.requireNonNull(files.listFiles())) {
       if (f.getName().endsWith(".jar")) {
         try {
+          log.info("Decompressing file: " + f.getName());
           JarUtils.decompressJarFile(outputDirectory, f.getAbsolutePath());
           // delete the original dependency jar file
           org.apache.commons.io.FileUtils.forceDelete(f);
         } catch (IOException e) {
-          log.warn("Problem decompressing jar file: " + f.getAbsolutePath());
+          log.error("Problem decompressing jar file: " + f.getAbsolutePath(), e);
```

Comment:
Changed to error instead of warn, open for suggestions, also added the exception. I know is not ideal to bubble an exception for the user, so open for feedback here.